network-api/networkapi/templates/fragments/donate_banner.html

```diff
@@ -26,7 +26,7 @@
                             {{ banner.subtitle }}
                         </p>
                         <div class="tw-w-full tw-pt-6">
-                            <a class="{{ btn_class }}" href="{{ banner.cta_link }}">
+                            <a id="banner-cta-button" class="{{ btn_class }}" href="{{ banner.cta_link }}">
```

Comment:
ID added for JS event tracking purposes

Reply:
Totally fine with using the `id` here for event tracking. But what came to mind is that we could instead use a custom data attribute to maintain a better separation of concerns and lower the risk of breaking an event. And I recall there being a ticket about this... but I can't seem to find it. I remember we determined it was a large undertaking to rewrite all the ga events and may even do more harm than good by accidentally breaking something. Perhaps something more to keep in mind for redesign as we start fresh and can think of a standard attribute naming convention.

Reply:
Thanks for the comments. I'm just reviewing the changes and all the context deeper, and agree with the data attributte to avoid any further issues or confusion with the events. Also regarding GTM events we can add `window.dataLayer.push({event: "event-name"});` and trigger the event inside of the current event listener `ctaButton?.addEventListener('click', (e) =>{});` where the GTM "event-name" will only be related to the way is shown in GA.

Reply:
Thanks y'all  for the discussion and feedback!

After reading through the conversation, I agree that using a custom data attribute could be beneficial in the future, especially once we establish a standard naming convention as part of the redesign. However, for now, I’ll go with Mavis' suggestion and continue using the ID attribute, since it's already being referenced in our existing JS code.

Thanks!